test.js

```diff
@@ -38,75 +38,165 @@ const inProgressReducer = (state = {}, action) => {
     }
 }
 
-const fetchUsersReducer = createMachine({
-    'INIT': initReducer,
-    'IN_PROGRESS': inProgressReducer
-})
+
+const initReducerWithCustomStatus = (state = {}, action) => {
+    switch (action.type) {
+    case 'FETCH_USERS':
+        return Object.assign({}, state, {
+            error: null,
+            my_custom_status: 'IN_PROGRESS'
+    })
+    default:
+        return state
+    }
+}
+
+const inProgressReducerWithCustomStatus = (state = {}, action) => {
+
+    switch (action.type) {
+    case 'FETCH_USERS_RESPONSE':
+        return Object.assign({}, state, {
+            error: null,
+            users: action.payload.users,
+            my_custom_status: 'INIT'
+        })
+    case 'FETCH_USERS_FAIL':
+        return Object.assign({}, state, {
+            error: action.payload,
+            my_custom_status: 'INIT'
+        })
+    default:
+        return state
+    }
+}
+
 
 // END FIXTURES
 
 // BEGIN HELPERS
 
-let state = undefined
-let prevState = undefined
+const createDummyStore = function (reducer, initialState) {
```

Comment:
`createDummyStore` creates an object that does not conform to the Redux store API: that is, its `dispatch` has a different signature and it is missing `getState`.  This was actually confusing for me when I tried to `console.log(storeWithCustomStatus.getState())`.